pallets/pools/src/lib.rs

```diff
@@ -705,6 +797,15 @@ pub mod pallet {
 				Error::<T>::InSubmissionPeriod
 			);
 
+			// Both changes.tranches and changes.tranche_metadata
+			// have to be NoChange or Change, we don't allow to change either or
+			// ^ = XOR, !^ = negated XOR
+			ensure!(
+				!((changes.tranches == Change::NoChange)
+					^ (changes.tranche_metadata == Change::NoChange)),
+				Error::<T>::UpdatePrerequesitesNotFulfilled
+			);
+
```

Comment:
👍 

Not sure about the `UpdatePrerequesitesNotFulfilled` error name tho. Maybe `UnmetUpdatePrerequesite`?

Reply:
I can change it. I just used the pre-existing error name `UpdatePrerequesitesNotFulfilled` since I thought it is getting confusing to have slightly different named errors.